media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayer = window.dataLayer = window.dataLayer || [];
 
-        dataLayer.push({
+    function sendCoreDataLayer() {
+        var dataLayerCore = {
             'event': 'core-datalayer-loaded',
-            'pageId': Mozilla.Analytics.getPageId()
-        });
+            'pageId': Mozilla.Analytics.getPageId(),
+            'pageHasDownload': Mozilla.Analytics.pageHasDownload(),
+            'pageHasVideo': Mozilla.Analytics.pageHasVideo(),
+            'pageVersion': Mozilla.Analytics.getPageVersion(),
+            'testPilotUser': window.navigator.testpilotAddon ? window.navigator.testpilotAddon : 'false'
```

Comment:
Could we maybe use `'testpilotAddon' in navigator ? window.navigator.testpilotAddon : false` instead? Also any reason we you're using a string for `'false'` at the end instead of a Boolean? (the property seems to return the latter from what I can see).